modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -109,6 +110,7 @@ const renderCommonStructure = (icon: Option<string>, text: Option<string>, toolt
     ]),
 
     eventOrder: {
+      [SystemEvents.receive()]: [ 'receiving', 'reflecting' ],
```

Comment:
This seems as though it should probably be in alloy instead of TinyMCE? given they are both default behaviours in alloy so you'd need to specify this in all places which is weird.

- [ ] Consider moving this to Alloy instead